daisy_workflows/image_import/inspection/tests/model_tests.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
```

Comment:
What does "model" mean in this context? Maybe add docs

Reply:
For "model", I mean the public-facing types that a caller can interact with; they're all defined in the `inspectors.model` package. I'm not wedded to the term "model", though. Any ideas for something that would be clearer?